nordicsemi/dfu/dfu_transport_mesh.py

```diff
@@ -307,10 +309,20 @@ def send_firmware(self, firmware):
         # Send firmware packets
         self.temp_progress = 0.0
         for (count, pkt) in enumerate(frames):
+            # First resend any requested packets
+            while len(self.requested_packets) > 0:
+                self.send_packet(SerialPacket(self.requested_packets.popleft()))
+                time.sleep(self.interval)
+            # Then send next frame
             self.send_packet(SerialPacket(pkt))
             self.log_progress(100.0 / float(frames_count))
             time.sleep(self.interval)
 
+        # Wait for any final missing packet requests
+        time.sleep(4)
```

Comment:
This timeout value should have a constant at the top like the others. Also, perhaps 4 seconds is a little short?

Reply:
Yeah it might be. 10 seconds might be better? I'll change it to 10 seconds and add a constant 👍